tsconfig.json

```diff
@@ -1,72 +1,19 @@
 {
   "compilerOptions": {
-    /* Basic Options */
```

Comment:
This config is the same, largely, sans the comments; The only real change is `lib` of `es2015` to `es6`